### DISCUSSION TITLE
Documentation "Hello World" should be complete runnable program

### DISCUSSION THREAD

**Discussion by reedjones** [gleam-lang/gleam#3212] [Ideas & suggestions]
The "Hello World" introduction is an interactive guided tour, I would personally would like to have seen a "hello world" this is how you run a gleam program on your machine right now. The current introduction is kinda cryptic and mentions a "normal" Gleam project - I would expect the introduction to show a normal project. 

Here is a tiny program that prints out the text "Hello, Joe!". We'll explain how it works shortly.

> In a normal Gleam project

How would it look in a normal project? 

 this program would be run using the command gleam run on the command line, but here in this tour the program is compiled and run inside your web browser,

>  allowing you to try Gleam without installing anything on your computer.

What if I do want to try Gleam on my computer?

**Comment by lpil** [comment]
Hello! This information is at the end of the tour.

**Reply by reedjones** [reply]
Yeah that's what I mean, like maybe could be good idea to include a link to that section somewhere at the start of the tour or something?
For example look at equivalent intro/tour thingys here: [go hello world](https://go.dev/doc/tutorial/getting-started) and [elixir hello world](https://hexdocs.pm/elixir/introduction.html) For me personally when I think of a 'hello world' I basically want to see what it takes to go from 0 to a running program.

**Reply by lpil** [reply]
We have exactly the same as Go. Go's tour doesn't show a whole project, it's just a REPL. The Gleam equivalent of that document you linked is here https://gleam.run/writing-gleam/
